dvc/scm/git/__init__.py

```diff
@@ -133,15 +132,24 @@ def _get_gitignore(self, path):
 
         return entry, gitignore
 
-    @staticmethod
-    def _ignored(entry, gitignore_path):
-        if os.path.exists(gitignore_path):
-            with open(gitignore_path, "r") as fobj:
-                ignore_list = fobj.readlines()
-            return any(
-                filter(lambda x: x.strip() == entry.strip(), ignore_list)
-            )
-        return False
+    def _ignored(self, entry, gitignore_path):
+
+        # We want to check first if `entry` is already being ignored
+        # by the .gitignore file in the root dir.
+
+        entry = (
+            entry[1:] if entry[0] == "/" else entry
```

Comment:
```suggestion
            entry = entry.lstrip("/")
```

Reply:
Looks like it won't be needed if we just pass `path` to `_ignored()` instead of putting the pieces back together.

Reply:
hmm don't know what happened there haha